src/docs/gold_standard_slot_examples.md

```diff
@@ -0,0 +1,444 @@
+# Gold Standard MIxS LinkML Slots Examples
+
+## Introduction
+
+This page lists a range of 'gold standard' examples of slots that are used in the MIxS LinkML schema.
+
+There are examples of slots that cover a range of common types of data, including:
+
+- Basic slots (i.e., match exactly to a LinkML `range` type):
+  - [Integer](#integer) (numeric)
+  - [Float](#float) (numeric)
+  - [String](#string) (free text)
+  - [Fixed list](#fixed-list) (enumeration)
+  - [Boolean](#boolean)
+- Complex (i.e., common more complex `range` types that are more specific to MIxS):
+  - [Structured text (text only)](#structured-text-text-only) (text only)
+  - [Structured text (with unit, measurement)](#structured-text-numeric-with-unit-measurement) (numeric with units)
+  - [Structured text (ontology term)](#structured-text-ontology-term)
+  - [URL](#url) (or other link utilising a URL like structure)
+  - [Mixed type (free text or ontology)](#mixed-type) (either free text, or from an ontology)
+
+You can use these as templates on how to construct your own new slot to the highest quality level expected.
+
+## Overview
+
+Every MIxS LinkML slot should have at a minimum the following attributes:
+
+- [`description`](https://linkml.io/linkml/schemas/metadata.html#providing-descriptions): the description of what the metadata term is for
+- [`title`](https://linkml.io/linkml-model/latest/docs/title/): a short human readable 'title' for the slot
+- [`examples`](https://linkml.io/linkml-model/latest/docs/examples/): examples values demonstrating how the slot should be used
+- [`in_subset`](https://linkml.io/linkml-model/latest/docs/in_subset/): the section of the schema that the slot belongs based on a [fix list of MIxS categories](https://github.com/GenomicsStandardsConsortium/mixs/blob/609b0f567486f64cb7061246588d8006f87fa138/src/mixs/schema/mixs.yaml#L21-L26)
+  - Note: this system may be replaced in the near future!
+- [`keywords`](https://linkml.io/linkml-model/latest/docs/keywords/): useful keywords to allow grouping of related slots together
```

Comment:
When I introduced the keywords in 6.2, I didn't have any tools or policies for adding new keywords. And I still don't! So new terms could be added with no keywords, or with keywords that are similar to but different from the ones I used. Nothing is checking that yet. And we don't really have any easy tooling for retrieving all terms that are tagged with a keyword. That can be done with `SchemaView` loops, or by dumping MIxS to a tabular representation, but I don't think we have all agreed on the best way of doing that either.